src/__tests__/__snapshots__/node.test.ts.snap

```diff
@@ -400,6 +401,7 @@ Object {
       "scrollbar": "14px",
     },
     "zIndex": Object {
+      "1": 1,
```

Comment:
in the margin, we use `px` as the key for `1px`

Reply:
can you please check if this value is in the shared config already?

Reply:
> can you please check if this value is in the shared config already?

in what package is the shared tailwind config included? I checked https://github.com/carforyou/carforyou-configuration-pkg but there is no tailwind configuration.

Reply:
> in the margin, we use `px` as the key for `1px`

As far as I know, z-index has no `px` value?

Reply:
1. Yeah, your z-index naming makes sense. I didn't think hard enough

2. Sorry for causing confusion. I meant to reference to the [shared tailwind config in components-pkg](https://github.com/carforyou/carforyou-components-pkg/blob/master/src/tailwind/defaultConfig.ts). It is merged with project-/package-specific values here https://github.com/carforyou/carforyou-header-footer-pkg/blob/master/tailwind.config.js
Currently in this package, it seems there are no custom values.